iocore/cache/P_CacheVol.h

```diff
@@ -206,6 +206,12 @@ struct Vol : public Continuation {
   int dir_check(bool fix);
   int db_check(bool fix);
 
+  bool
+  evac_bucket_valid(off_t bucket)
+  {
+    return (bucket >= 0 && bucket < evacuate_size);
```

Comment:
How can an `off_t` be less than zero?

Reply:
The macro `dir_evac_bucket` produces a `long long` and seem to be stored as `off_t` in other places in the code so I thought it might be a reasonable type for the argument here.  `off_t` is a signed type so I check for >= 0.  Ideally the macro would return an unsigned bucket, but I'm not sure the implication of that change.